config/application.rb

```diff
@@ -17,6 +17,8 @@
 # you've limited to :test, :development, or :production.
 Bundler.require(*Rails.groups)
 
+Dotenv::Railtie.load
```

Comment:
From their docs I believe this loading is redundant however I think it will be helpful to have it and be explicit in what it's purpose is. Perhaps an additional comment along the lines of:

> For secret management in live environments only. Development ENV still managed through `docker-compose.env`

What do you think?